client/scripts/models/DiscussionDraft.ts

```diff
@@ -0,0 +1,40 @@
+import moment from 'moment-twitter';
+import { IUniqueId } from './interfaces';
+import OffchainAttachment from './OffchainAttachment';
+
+class DiscussionDraft {
+  public readonly id: number;
+  public readonly author: string;
+  public readonly authorChain: string;
+  public readonly title: string;
+  public readonly body: string;
+  public readonly attachments: OffchainAttachment[];
+  public readonly createdAt: moment.Moment;
+  // Draft tags are not created as tag objects yet, so they are just strings
+  public readonly tag: string;
+  public readonly community: string;
+  public readonly chain: string;
+  public readonly slug = 'draft';
+
+  constructor(
+    author: string,
```

Comment:
no authorChain?

Reply:
also, you're in the process of updating the association to addresses right? just making a note here to track along with the other thread items

Reply:
authorChain's been added

Re: addresses, yes, I wrote a migration to change author_id on threads/drafts to address_id. I haven't yet updated the author return value on route lookup/validation checks to assign to variable alternatively named `address`. I can try to fit that in though it seems low priority for time being?